traitsui/testing/tester/_ui_tester_registry/qt4/_control_widget_registry.py

```diff
@@ -32,6 +32,19 @@ def _handle_is_enabled(wrapper, interaction):
     return wrapper._target.control.isEnabled()
 
 
+def _handler_is_visible(wrapper, interaction):
```

Comment:
nit - conform to the existing naming convention
```suggestion
def _handle_is_visible(wrapper, interaction):
```